jbpm-console-ng-case-mgmt/jbpm-console-ng-case-mgmt-backend/src/test/java/org/jbpm/console/ng/cm/backend/server/RemoteCaseManagementServiceImplTest.java

```diff
@@ -174,4 +177,62 @@ public void testGetCaseInstance() throws Exception {
         assertCaseInstance(ci, cis);
     }
 
+    @Test
+    public void testGetCaseComments() throws Exception {
+        final CaseComment caseComment = new CaseComment();
+        caseComment.setId("CommentId");
+        caseComment.setAuthor("Author");
+        caseComment.setText("Text");
+        caseComment.setAddedAt(new Date());
+
+        when(caseServicesClient.getComments(anyString(), anyString(), anyInt(), anyInt())).thenReturn(
```

Comment:
I would suggest using the expected parameter values in here so that you can verify they're passed down correctly. anyString and anyInt could hide potential issues.